projects/assets/src/translations/en/common.ts

```diff
@@ -63,5 +63,38 @@ export const common = {
     forbidden: 'You are not authorized to perform this action.',
     gatewayTimeout: 'The server did not responded, please try again later.',
     unknownError: 'An unknown error occured',
+    validationErrors: {
```

Comment:
We prefer more flat structure like
```
validationErrors: {
  card_cardType: 'The credit card selected is not supported. Please select another.',
  card_accountNumber: 'The credit card number entered is not valid.',
  // ...
},
```
But if you decide to use reason as key, better will be schema validationErrors.missing.card_cardType

Reply:
Using params is better idea, but we have excepation (card_cardType) rest can use chunk like 
````
enteredValueNotValid: 'The {{param}} entered is not valid.'
````